_rules/focusable-no-keyboard-trap-80af7b.md

```diff
@@ -76,91 +76,120 @@ These [focusable][] elements do not create a trap for keyboard navigation.
 
 #### Passed Example 2
 
-This [focusable][] element does not create a trap for keyboard navigation.
+This element is made [focusable][] by the `tabindex` attribute. It does not create a trap for keyboard navigation.
 
 ```html
 <div tabindex="1">Text</div>
 ```
 
 #### Passed Example 3
 
-This [focusable][] element does not create a trap for keyboard navigation.
+This element is made [focusable][] by the `tabindex` attribute, even if it is not part of the sequential focus navigation. It does not create a trap for keyboard navigation.
 
 ```html
 <div tabindex="-1">Text</div>
 ```
 
 #### Passed Example 4
 
-These focusable `button` elements create a keyboard trap with help information in a paragraph before, and where the method advised works.
+These focusable `button` elements have scripts that create a keyboard trap. The document includes help information in a paragraph before the `button` elements and the method advised works to escape the keyboard trap.
 
 ```html
 <script>
 	var trapOn = false
+
+	const moveFocusToButton = btn => {
+		if (trapOn) {
+			document.getElementById(btn).focus()
+		}
+	}
+
+	const escapeTrapOnCtrlM = e => {
```

Comment:
These two functions are used in many examples in two rules and might benefit from a shared asset. Obviously with the drawback of less locality… (that might be solved by #1805 ).

Reply:
Hoping that #1805 will become a reality soon, I'm moving all scripts to test-assets